### ISSUE TITLE
Move this repository to github

### ISSUE THREAD

**Issue by GoogleCodeExporter** [oborel/obo-relations#41]
```
This can be done with "git svn"

Example in another repo:
https://code.google.com/p/owltools/issues/detail?id=111
```

Original issue reported on code.google.com by `cmung...@gmail.com` on 30 Jan 2015 at 12:57

**Comment by cmungall** [comment]
This is done, obviously, mostly.

todo: redirect purls. @jamesaoverton -- OBI etc use http://purl.obolibrary.org/obo/ro/core.owl we can switch this to pulling from github. I don't know if https is still an issue for folks?

what about versioned purls?

**Comment by jamesaoverton** [comment]
In the OBI development files, we recently started specifying dated PURLs for imports. We're currently using <http://purl.obolibrary.org/obo/ro/releases/2015-04-12/core.owl>, and that's the only dated RO PURL we've used so far.

When releasing OBI we merge everything into a single release file.

Last I checked, the SSL issue is still unresolved in Protege 4. I've heard from several people that they can't use Protege 5, but I don't know the details for each case. So I think that HTTPS is still a problem.
